extensions/amp-autocomplete/0.1/amp-autocomplete.js

```diff
@@ -470,6 +473,26 @@ export class AmpAutocomplete extends AMP.BaseElement {
     }
   }
 
+  /**
+   * If a fallback element is provided, displays it instead of suggestions.
+   * Otherwise, throws given error. Must be called in a mutate context.
+   * @param {*=} error
+   * @throws {!Error} If fallback element is not present.
+   * @private
+   */
+  renderFallbackUI_(error) {
+    this.clearAllItems_();
+    const fallback = childElementByAttr(this.element, 'fallback');
+    if (fallback) {
```

Comment:
Can use `toggleFallback` directly? https://github.com/ampproject/amphtml/blob/master/src/custom-element.js#L1509

Reply:
+1

Reply:
`toggleFallback` does show and hide the `[fallback]` element, but it feels wrong because it applies the `.amp-notsupported` className which doesn't make sense in this context. But maybe it's not a concern if users don't use it to apply styles?